### PR TITLE
feat: upgrade towards net6.0

### PR DESCRIPTION
**Pull request by stijnmoreels**
Upgrade packages towards .NET 6.

Relates to https://github.com/arcus-azure/arcus/issues/188

### CODE REVIEW

File: src/Arcus.Testing.Logging/Arcus.Testing.Logging.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFrameworks>net6.0;netcoreapp3.1;netstandard2.1</TargetFrameworks>
```
Comment (fgheysels) [inline src/Arcus.Testing.Logging/Arcus.Testing.Logging.csproj:4]: Is it correct to still have .NET Core 3.1 here ?

File: src/Arcus.Testing.Security.Providers.InMemory/Arcus.Testing.Security.Providers.InMemory.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFrameworks>net6.0;netcoreapp3.1;netstandard2.1</TargetFrameworks>
```
Comment (fgheysels) [inline src/Arcus.Testing.Security.Providers.InMemory/Arcus.Testing.Security.Providers.InMemory.csproj:4]: Is it correct to still have .NET Core here ?

Comment (fgheysels) [reply src/Arcus.Testing.Security.Providers.InMemory/Arcus.Testing.Security.Providers.InMemory.csproj:4]: Maybe we should just consider supporting .NET 6 then ?

Comment (stijnmoreels) [reply src/Arcus.Testing.Security.Providers.InMemory/Arcus.Testing.Security.Providers.InMemory.csproj:4]: Ah, that's a good one. Yeah, maybe we shouldn't even concern ourselves with backwards compatibility here. Good one. 👍

Comment (stijnmoreels) [reply src/Arcus.Testing.Security.Providers.InMemory/Arcus.Testing.Security.Providers.InMemory.csproj:4]: I'm wondering, though, we mention this in-memory secret provider as a way of easily testing your secret store setup in the feature docs of the security repo... maybe it's not a bad idea to have it still support .NET Core, for a while.

Comment (fgheysels) [reply src/Arcus.Testing.Security.Providers.InMemory/Arcus.Testing.Security.Providers.InMemory.csproj:4]: Ok for me.

**Review by fgheysels — APPROVED** [review]

**Pull request merged**